plugins/modules/net_tools/nmcli.py

```diff
@@ -754,6 +756,66 @@
                     - The username used to authenticate with the network, if required.
                     - Many providers do not require a username, or accept any username.
                     - But if a username is required, it is specified here.
+    wireguard:
+        description:
+            - The configuration of the Wireguard connection.
+            - Note the list of suboption attributes may vary depending on which version of NetworkManager/nmcli is installed on the host.
+            - 'An up-to-date list of supported attributes can be found here:
+              U(https://networkmanager.dev/docs/api/latest/settings-wireguard.html).'
+            - 'For instance to configure a listen port:
+              C({listen-port: 12345}).'
+        type: dict
+        version_added: 4.3.0
+        suboptions:
+            fwmark:
+                description:
+                    - The 32-bit fwmark for outgoing packets.
+                    - The use of fwmark is optional and is by default off. Setting it to 0 disables it.
+                    - Note that C(ip4-auto-default-route) or C(ip6-auto-default-route) enabled, implies to automatically choose a fwmark.
+                type: int
+                default: 0
```

Comment:
```suggestion
```
The default values come from `nmcli` and vary with versions so I think it's better not to mention here.

Reply:
Thanks for reviewing! All suggestions applied. Didn't think about potential different defaults from nmcli, makes sense.